From stable-bounces@linux.kernel.org  Fri Dec  1 19:42:43 2006
Date: Sat, 2 Dec 2006 14:37:27 +1100
To: stable@kernel.org
Message-ID: <20061202033726.GA26175@gondor.apana.org.au>
From: Herbert Xu <herbert@gondor.apana.org.au>
Subject: cryptoloop: Select CRYPTO_CBC

As CBC is the default chaining method for cryptoloop, we should select
it from cryptoloop to ease the transition.

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
---
 drivers/block/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.19.orig/drivers/block/Kconfig
+++ linux-2.6.19/drivers/block/Kconfig
@@ -305,6 +305,7 @@ config BLK_DEV_LOOP
 config BLK_DEV_CRYPTOLOOP
 	tristate "Cryptoloop Support"
 	select CRYPTO
+	select CRYPTO_CBC
 	depends on BLK_DEV_LOOP
 	---help---
 	  Say Y here if you want to be able to use the ciphers that are 
